pixelgl/gltriangles.go

```diff
@@ -101,15 +101,16 @@ func (gt *GLTriangles) updateData(t pixel.Triangles) {
 	}
 
 	// TrianglesData short path
+	s := gt.vs.Stride()
+	l := gt.Len()
```

Comment:
Use longer variable names here. These variables are used in a larger context than 5 lines of code, so this is hurts readability.